kinto/config/kinto.tpl

```diff
@@ -11,49 +11,131 @@ port = %(http_port)s
 [app:main]
 use = egg:kinto
 
+# Production settings
+#
+# https://kinto.readthedocs.io/en/latest/configuration/production.html
+#
+# kinto.statsd_backend = kinto.core.statsd
+# kinto.statsd_url = udp://localhost:8125
+# kinto.statsd_prefix = kinto-prod
+
+# kinto.http_scheme = https
+# kinto.http_host = kinto.services.mozilla.com
+
+# kinto.backoff = 10
+# kinto.retry_after_seconds = 30
+# kinto.eos =
+
+# Full options list for .ini file
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html
+
+# Feature settings
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#feature-settings
 #
-# Backends.
+# kinto.readonly = False
+# kinto.batch_max_requests = 25
+# kinto.paginate_by=None
+# kinto.<object-type>_id_generator=UUID4 <bucket, collection, group, record>
```

Comment:
This is super advanced configuration, we could move it at the end in a dedicated section. I think it would be more readable with a line per resource. Plus, the default id generator is something like `kinto.views.NameGenerator`